### DISCUSSION TITLE
Show Top bar on iphone but not on tablet?

### DISCUSSION THREAD

**Discussion by beeptech13** [NemesisRE/kiosk-mode#142] [Q&A]
This is a great add-on. Is there a way to show the top bar on an iphone but not on tablet?

**Comment by elchininet** [comment]
Hi @beeptech13,
You can use [user settings](https://github.com/NemesisRE/kiosk-mode#user_settings) for that, a set of settings for the user on the phone and another settings for the user on the tablet.
Regards

**Comment by beeptech13** [comment]
Thanks.  That worked perfectly.
